cmd/qat_plugin/qat_plugin.go

```diff
@@ -210,7 +210,8 @@ func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
 	for _, driver := range append(dp.kernelVfDrivers, dp.dpdkDriver) {
 		files, err := ioutil.ReadDir(path.Join(dp.pciDriverDir, driver))
 		if err != nil {
-			return nil, fmt.Errorf("Can't read sysfs for driver %s: %+v", driver, err)
```

Comment:
I thought about this (I noticed it triggers if the drivers parameter is not passed and the driver I have comes later in the default string) and the message in my earlier review. It'd probably be fair to tell the driver/device will be ignored/skipped now. How about simply:  ```Driver %s not available. Skipping.```

Reply:
Done!!

Reply:
Sorry, missed the unit tests. Will update.